### PR TITLE
[GCP] Convert GCP GKE metricset from lightweight

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Convert GCP GKE metricset from lightweight

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist

- [ ]

## Related issues

- Relates #490
- Requires #2301
- Requires #3798

### CODE REVIEW

File: packages/gcp/_dev/build/docs/metrics.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Google Cloud Platform Metrics Integration
+
+The Google Cloud Platform Metrics integration collects and parses Google Cloud billing, compute, storage metrics.
+
+
```
Comment (efd6) [inline packages/gcp/_dev/build/docs/metrics.md:5]:
```suggestion
```

File: packages/gcp/_dev/build/docs/gke.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Google Kubernetes Engine (GKE)
+
+## Metrics
+
+This is the `gke` dataset.
```
Comment (efd6) [inline packages/gcp/_dev/build/docs/gke.md:5]: Can this be fleshed out a little? Maybe link to https://cloud.google.com/monitoring/api/metrics_kubernetes as well?

File: packages/gcp/kibana/dashboard/gcp-1ae960c0-f9f8-11eb-bc38-79936db7c106.json [hunk 3]
```diff
@@ -0,0 +1,3119 @@
+{
```
[the remark this answers is not in the document]
Comment (legoguy1000) [reply packages/gcp/kibana/dashboard/gcp-1ae960c0-f9f8-11eb-bc38-79936db7c106.json:1]: Not with any data in it. It's just the original dashboards/visualizations from metricbeat.

[the remark this answers is not in the document]
Comment (efd6) [reply packages/gcp/kibana/dashboard/gcp-1ae960c0-f9f8-11eb-bc38-79936db7c106.json:1]: No worries. Thanks.

**Comment by legoguy1000** [comment]
@kaiyan-sheng ready to merge this once #3798 is merged.

**Comment by legoguy1000** [comment]
@kaiyan-sheng ready for test.

**Review by kaiyan-sheng — APPROVED** [review]

**Pull request merged**